### ISSUE TITLE
Partners marked for deletion are showing in the partnership dropdown and should be hidden

### ISSUE THREAD

**Issue by anusenan** [unicef/etools-issues#64]

**Comment by anusenan** [comment]
![image](https://cloud.githubusercontent.com/assets/25492170/23724601/57ce2228-041b-11e7-91e8-4c4cb6a6e7e3.png)

**Comment by robertavram** [comment]
@anusenan I think this was done in Trello. please update

**Comment by aldohunicef** [comment]
![image](https://cloud.githubusercontent.com/assets/26142645/24964879/79d21878-1f70-11e7-83c1-ae3a16715ea8.png)

**Comment by aldohunicef** [comment]
![image](https://cloud.githubusercontent.com/assets/26142645/24964955/a5685588-1f70-11e7-8d27-dfa504b8f4e7.png)

**Comment by aldohunicef** [comment]
![image](https://cloud.githubusercontent.com/assets/26142645/24965209/681cdce8-1f71-11e7-9566-7545afc89008.png)

**Comment by aldohunicef** [comment]
April, 12 test results indicate that the hidden partners are still appearing in the drop down menu. In some cases there are more than 12 hidden partners so the gap or empty space is covering the whole drop down menu visible area.

**Comment by aldohunicef** [comment]
@janpanchalingam1 Please proceed and create a new card or amend this one

**Comment by lordfuoco** [comment]
@janpanchalingam1 @anusenan @torbent   this card is a bit obsolete, we need to alter a bit an existing API endpoint on the backend to be able to hook it up on the frontend and solve this issue.
Should we have new cards or should we go on in this one ?

**Comment by janpanchalingam1** [comment]
@lordfuoco are you able to continue on this card?

**Comment by torbent** [comment]
@janpanchalingam1 @anusenan this is pending on #334 then we can continue here

**Comment by janpanchalingam1** [comment]
Passed!
